includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1841,7 +1841,7 @@ public function generate_upe_checkout_experience_accepted_payments_html( $key, $
 		foreach ( $this->payment_methods as $method_id => $method ) {
 			$method_enabled       = in_array( $method_id, $this->get_upe_enabled_payment_method_ids(), true ) && ( $is_automatic_capture_enabled || ! $method->requires_automatic_capture() ) ? 'enabled' : 'disabled';
 			$method_enabled_label = 'enabled' === $method_enabled ? __( 'enabled', 'woocommerce-gateway-stripe' ) : __( 'disabled', 'woocommerce-gateway-stripe' );
-			$capability_id        = "{$method_id}_payments"; // "_payments" is a suffix that comes from Stripe API, except when it is "transfers", which does not apply here
+			$capability_id        = WC_Stripe_Helper::get_payment_method_capability_id( $method_id ); // "_payments" is a suffix that comes from Stripe API, except when it is "transfers", which does not apply here
```

Comment:
The comment here doesn’t seem relevant anymore. Maybe we should move it to the `get_payment_method_capability_id` you just created.

Reply:
Good call, moved it in [ba77fd5](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4073/commits/ba77fd51715c1529b1353322e699ba7ceaab89a4)